tests/scripts/arm32_ci_script.sh

```diff
@@ -214,8 +214,8 @@ function cross_build_coreclr {
     source "$__ARMEmulRootfs"/dotnet/setenv/setenv_incpath.sh "$__ARMEmulRootfs"
 
     #Apply the changes needed to build for the emulator rootfs
-    (set +x; echo 'Applying cross build patch to suit Linux ARM emulator rootfs')
-    git am < "$__ARMEmulRootfs"/dotnet/setenv/coreclr_cross.patch
+    #(set +x; echo 'Applying cross build patch to suit Linux ARM emulator rootfs')
```

Comment:
If this is no longer needed, could you please delete all the commented out lines instead?